src/Polly.Core.Benchmarks/README.md

```diff
@@ -12,19 +12,19 @@ LaunchCount=2  WarmupCount=10
-|                    |            |             |           |            |             |       |         |        |           |             |
-| ExecutePipeline_V7 |          2 |   165.37 ns |  1.157 ns |   1.732 ns |   165.59 ns |  1.00 |    0.00 | 0.0658 |     552 B |        1.00 |
-| ExecutePipeline_V8 |          2 |   119.10 ns |  0.653 ns |   0.915 ns |   119.63 ns |  0.72 |    0.01 |      - |         - |        0.00 |
-|                    |            |             |           |            |             |       |         |        |           |             |
-| ExecutePipeline_V7 |          5 |   533.97 ns |  7.327 ns |  10.967 ns |   536.79 ns |  1.00 |    0.00 | 0.1545 |    1296 B |        1.00 |
-| ExecutePipeline_V8 |          5 |   227.69 ns |  1.236 ns |   1.812 ns |   227.72 ns |  0.43 |    0.01 |      - |         - |        0.00 |
-|                    |            |             |           |            |             |       |         |        |           |             |
-| ExecutePipeline_V7 |         10 | 1,191.41 ns | 35.512 ns |  53.152 ns | 1,192.79 ns |  1.00 |    0.00 | 0.3014 |    2536 B |        1.00 |
-| ExecutePipeline_V8 |         10 |   557.95 ns | 76.434 ns | 112.036 ns |   505.58 ns |  0.47 |    0.09 |      - |         - |        0.00 |
+|             Method | Components |        Mean |      Error |      StdDev | Ratio | RatioSD |   Gen0 | Allocated | Alloc Ratio |
+|------------------- |----------- |------------:|-----------:|------------:|------:|--------:|-------:|----------:|------------:|
+| ExecutePipeline_V7 |          1 |    88.27 ns |   8.170 ns |    11.72 ns |  1.00 |    0.00 | 0.0362 |     304 B |        1.00 |
+| ExecutePipeline_V8 |          1 |   194.33 ns |  12.771 ns |    17.90 ns |  2.25 |    0.31 |      - |         - |        0.00 |
+|                    |            |             |            |             |       |         |        |           |             |
+| ExecutePipeline_V7 |          2 |   229.53 ns |   7.011 ns |    10.28 ns |  1.00 |    0.00 | 0.0658 |     552 B |        1.00 |
+| ExecutePipeline_V8 |          2 |   387.62 ns |   8.655 ns |    11.85 ns |  1.69 |    0.08 |      - |         - |        0.00 |
+|                    |            |             |            |             |       |         |        |           |             |
+| ExecutePipeline_V7 |          5 |   698.12 ns |  14.359 ns |    21.05 ns |  1.00 |    0.00 | 0.1545 |    1296 B |        1.00 |
+| ExecutePipeline_V8 |          5 |   871.83 ns | 168.409 ns |   224.82 ns |  1.25 |    0.31 |      - |         - |        0.00 |
+|                    |            |             |            |             |       |         |        |           |             |
+| ExecutePipeline_V7 |         10 | 2,255.86 ns | 735.997 ns | 1,078.81 ns |  1.00 |    0.00 | 0.3014 |    2536 B |        1.00 |
+| ExecutePipeline_V8 |         10 | 1,455.29 ns |  52.188 ns |    73.16 ns |  0.78 |    0.33 |      - |         - |        0.00 |
```

Comment:
This change seems quite impactful in terms of relative time, though V7 is slower too so I guess it's just noise?

Reply:
v8:v7 performance ratio got much worse with this change. We are already looking how to address this.